src/proto/Solana.proto

```diff
@@ -22,16 +22,31 @@ message WithdrawStake {
     uint64 value = 2;
 }
 
+message CreateTokenAccount {
+    string token_mint_address = 1;
+    string token_address = 2;
+}
+
+message TokenTransfer {
+    string token_mint_address = 1;
+    string sender_token_address = 2;
+    string recipient_token_address = 3;
+    uint64 amount = 4;
```

Comment:
is `uint64` large enough?

Reply:
yes, that's how it's in solana code; plus there's only 8 bytes room in the encoded data.